src/index.js

```diff
@@ -0,0 +1,26 @@
+// assumes the resize-observer-polyfill is loaded, this can be done through polyfill.io
+export function useElementSize(elementRef) {
+  const [size, setSize] = React.useState({ width: 0, height: 0 })
+  const observerRef = React.useRef(null)
+
+  React.useEffect(() => {
+    // @ts-ignore
+    observerRef.current = new window.ResizeObserver(callback)
```

Comment:
Uit interesse: de callback staat niet in je dependencies, is die linter nu zo slim dat hij ziet dat in callback enkel een setSize voorkomt en daarom niet nodig is in de deps?

Reply:
Sidenote: ik krijg nog geen eslint error in de libraries, wellicht dat we de configfiles (en die voor stylelint) nog even aan de libraries toe moeten voegen

Reply:
Ja, lijkt me wel de bedoeling. Heb ik bij deze gedaan.
Krijg echter nog steeds geen linting warning voor de missende callback dependency